sdk/eventhub/azure-eventhubs/azure/eventhub/_eventprocessor/_eventprocessor_mixin.py

```diff
@@ -3,26 +3,39 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+import six
+import datetime
 from contextlib import contextmanager
 
 from azure.core.tracing import SpanKind  # type: ignore
 from azure.core.settings import settings  # type: ignore
 
-from azure.eventhub import EventPosition
+from azure.eventhub._common import EventPosition
```

Comment:
I thought this object was removed?

Reply:
I'm removing it right now, will update soon